src/components/EditorWrapper.vue

```diff
@@ -182,7 +182,7 @@ export default {
 			return this.$store.state.showAuthorAnnotations
 		},
 		lastSavedStatus() {
-			return this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
+			return this.hasConnectionIssue ? t('text', 'Offline, changes will be saved when online') : this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
```

Comment:
Small suggest to improve readability of the code:

```suggestion
			if (this.hasConnectionIssue) {
				return t('text', 'Connection lost. Will be saved once connection is restored.')
			}
			return this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
```

Otherwise good to go with the new wording from my side.

Reply:
Now the logic to check for `this.hasConnectionIssue` is doubled, looks like that was accidently @luka-nextcloud  :grimacing: